engine/runtime-parser/src/main/scala/org/enso/compiler/core/ir/module/scope/definition/Method.scala

```diff
@@ -49,13 +49,34 @@ object Method {
     */
   sealed case class Explicit(
     override val methodReference: Name.MethodReference,
-    override val body: Expression,
+    val bodyList: Seq[Expression],
```

Comment:
Using `Seq[Expression]` (even there is always a single expression) makes the `bodyList` subtree __lazy__. When `Method.Explicit` is deserialized from `Persistance.Input`, it doesn't resolve its `bodyList` (neither `lazy val body`) which __tremendously lowers__ the amount of _deserialized objects_ that need to be created in the memory.